### PR TITLE
Fix ucfs claimant lambda

### PR DESCRIPTION
**Pull request by connorIW**

### CODE REVIEW

**Review by steveburton4 — APPROVED** [review]

File: ucfs-claimant-api-mysql-interface.tf [hunk 1]
```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
     command = "./initial-commit.sh ${github_repository.ucfs_claimant_api_mysql_interface.name} '${github_repository.ucfs_claimant_api_mysql_interface.description}' ${github_repository.ucfs_claimant_api_mysql_interface.template.0.repository}"
   }
 }
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_email" {
```
Comment (dan-hill2802) [inline ucfs-claimant-api-mysql-interface.tf:58]: resources with underscores please

Comment (steveburton4) [reply ucfs-claimant-api-mysql-interface.tf:58]: Good point, my eyes did not pick this up 😂

File: ucfs-claimant-api-mysql-interface.tf [hunk 2]
```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
...
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_USERNAME"
+  plaintext_value = var.github_username
+}
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_token" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_TOKEN"
+  plaintext_value = var.github_token
+}
```
Comment (dan-hill2802) [inline ucfs-claimant-api-mysql-interface.tf:74]: this isn't needed/used

File: ucfs-claimant-api-mysql-interface.tf [hunk 3]
```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
     command = "./initial-commit.sh ${github_repository.ucfs_claimant_api_mysql_interface.name} '${github_repository.ucfs_claimant_api_mysql_interface.description}' ${github_repository.ucfs_claimant_api_mysql_interface.template.0.repository}"
   }
 }
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_email" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_EMAIL"
```
Comment (dan-hill2802) [inline ucfs-claimant-api-mysql-interface.tf:60]:
```suggestion
  secret_name     = "CI_GITHUB_EMAIL"
```

File: ucfs-claimant-api-mysql-interface.tf [hunk 4]
```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
...
+  plaintext_value = var.github_email
+}
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_username" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_USERNAME"
```
Comment (dan-hill2802) [inline ucfs-claimant-api-mysql-interface.tf:66]:
```suggestion
  secret_name     = "CI_GITHUB_USERNAME"
```

File: ucfs-claimant-api-mysql-interface.tf (hunk 2 above)
Comment (connorIW) [reply ucfs-claimant-api-mysql-interface.tf]: May be required in the future. I'd argue we should supply these secrets to all repos if they can't be accessed externally.

**Review by steveburton4 — APPROVED** [review]

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**